var/spack/repos/builtin/packages/kaldi/package.py

```diff
@@ -43,7 +44,9 @@ class Kaldi(Package):  # Does not use Autotools
     depends_on("cub", when="@2019-07-29:^cuda@:10")
 
     patch("openfst-1.4.1.patch", when="@2015-10-07")
-    patch("0001_CMakeLists_txt.patch", when="+cuda@11:")
+    patch("0001_CMakeLists_txt.patch", when="@:2021-11-16")
+    patch("0001-Fix-CUDA_INCLUDE_DIRS.patch", when="@2023-11-13:")
+    patch("0002-configure-Fix-gcc-version-check-for-cuda.patch", when="@2023-11-13:")
```

Comment:
Should the conditions on these patches include `+cuda`, especially since it is disabled by default?

Reply:
> Should the conditions on these patches include `+cuda`, especially since it is disabled by default?

Again, I would opt for 'no'. Simplicity ...